### PR TITLE
added ndc nomination to featured components

### PR DESCRIPTION
**Pull request by codingshot**
added ndc nomination under featured components under devhub

### CODE REVIEW

File: src/FeaturedComponents.jsx [hunk 1]
```diff
@@ -62,6 +62,12 @@ return (
           src="${REPL_ACCOUNT}/widget/ComponentCard"
           props={{ src: "devgovgigs.near/widget/Ideas" }}
         />
+      </Item>
+            <Item>
+        <Widget
+          src="${REPL_ACCOUNT}/widget/ComponentCard"
+          props={{ src: "nomination.ndctools.near/widget/NDC.Nomination.Page" }}
+        />
```
Comment (shelegdmitriy) [inline src/FeaturedComponents.jsx:70]:
```suggestion
      </Item>
      <Item>
        <Widget
          src="${REPL_ACCOUNT}/widget/ComponentCard"
          props={{ src: "nomination.ndctools.near/widget/NDC.Nomination.Page" }}
        />
```

**Review by shelegdmitriy — CHANGES_REQUESTED** [review]

File: src/FeaturedComponents.jsx [hunk 2]
```diff
@@ -62,6 +62,12 @@ return (
           src="${REPL_ACCOUNT}/widget/ComponentCard"
           props={{ src: "devgovgigs.near/widget/Ideas" }}
         />
+      </Item>
+            <Item>
+        <Widget
+          src="${REPL_ACCOUNT}/widget/ComponentCard"
+          props={{ src: "nomination.ndctools.near/widget/NDC.Nomination.Page" }}
```
Comment (charleslavon) [inline src/FeaturedComponents.jsx:69]: @codingshot do you have a testnet version of this component? If so, please commit an update to the replacements.*.json files as mentioned in [this PR](https://github.com/near/near-discovery-components/pull/167#discussion_r1293855988) which you can use as a reference.

**Comment by mpeterdev** [comment]
closing since a simpler process for adding featured components will be landing soon with #183

**Comment by TiffanyGYJ** [comment]
@mpeterdev NDC widget adding is time sensitive as the nomination and voting will be within this month, if this update for pull request by code/shot suffice our need, can we merge this for now?

**Comment by mpeterdev** [comment]
> @mpeterdev NDC widget adding is time sensitive as the nomination and voting will be within this month, if this update for pull request by code/shot suffice our need, can we merge this for now?

This PR could be merged, but it conflicts with the changes made in #183 so it would be ideal not to merge it. The only thing we are waiting on to roll #183 to prod is for our management account for be funded by Pagoda bizops so we can ping them to move it along

**Pull request closed without merging**